lib/rubocop/cop/rspec/rails/minitest_assertions.rb

```diff
@@ -22,30 +24,23 @@ module Rails
         #   expect(b).not_to eq(a)
         #   expect(a).to eq(nil)
         #   expect(a).not_to be_empty
+        #   expect(a).to be_truthy
+        #   expect(b).to be_falsey
```

Comment:
`assert_true` and `assert_false` are methods in Test::Unit and don't exist in Minitest (and ActiveSupport::TestCase):

- Minitest ... https://docs.seattlerb.org/minitest/Minitest/Assertions.html
- Test::Unit ... https://test-unit.github.io/test-unit/ja/Test/Unit/Assertions.html

 Should these be supported in `RSpec/Rails/Minitest`?

If so, `assert_true` should be contrasted with `be(true)`, not `be_truthy`. And `assert_false` should be contrasted with `be(false)`, not `be_falsey`.

Reply:
The relationship between test::unit and minitest assertions is not trivial https://github.com/rspec/rspec-core/blob/f8c8880dabd8f0544a6f91d8d4c857c1bd8df903/lib/rspec/core/test_unit_assertions_adapter.rb#L9

The cop, despite the name, is taking care of all of those assertions.